cupy/cusparse.py

```diff
@@ -73,19 +73,6 @@ def _call_cusparse(name, dtype, *args):
     return f(*args)
 
 
-def _dtype_to_DataType(dtype):
```

Comment:
This function is called also from `cupyx/scipy/sparse/linalg/_eigen.py` and `cupyx/scipy/sparse/linalg/_iterative.py`. Could you fix them?

Reply:
Thanks, @asi1024! Should be fixed in e4bede5.